### DISCUSSION TITLE
Can others get a Galaxy Brain Badge from answering this post p1?

### DISCUSSION THREAD

**Discussion by peterdanwan** [peterdanwan/gimme_readme#19] [Q&A] [answered]
Can others get a Galaxy Brain Badge from answering this post? (This is to test GitHub's features).

**Comment by Mimi-Dang** [comment, marked as answer]
It is possible if you put my answer as the right answer.

**Reply by peterdanwan** [reply]
Let's test that theory! I will open one more Q&A for you to respond to.

**Reply by Mimi-Dang** [reply]
Thank you @peterdanwan :)
